README.md

```diff
@@ -1334,6 +1334,80 @@ Reset the location service on real device since Appium 1.22.0.
 It could delay a few seconds to reflect the location by the system.
 It raises an error if the device is simulator or an error occurred during the reset.
 
+### mobile: enableConditionInducer
+
+Important: Device conditions are available for real devices running iOS 13.0 and later.
+
+Please mention this method will throw an error if you try to start an inducer without disabling the previous one
```

Comment:
Sorry, for not being clear on that. The note itself was a hint for you to add this explanation rather that just copypasting it into readme.

The string for the REDME would look like:

This API is going to throw an error if it is called while another condition inducer has been already enabled and is not explicitly disabled.

Reply:
Sorry, it's my mistake. I am learning to improve my English level